pyanaconda/ui/gui/spokes/eula.py

```diff
@@ -0,0 +1,103 @@
+import logging
+
+from pyanaconda.ui.common import FirstbootOnlySpokeMixIn
+from pyanaconda.ui.gui.spokes import NormalSpoke
+from pyanaconda.core.i18n import _, CN_
+from pyanaconda.core import eula
+from pyanaconda.ui.categories.eula import LicensingCategory
+from pyanaconda.anaconda_loggers import get_module_logger
+from pykickstart.constants import FIRSTBOOT_RECONFIG
+
+log = get_module_logger(__name__)
+__all__ = ["EULASpoke"]
+
+
+class EULASpoke(FirstbootOnlySpokeMixIn, NormalSpoke):
```

Comment:
`FirstbootOnlySpokeMixIn` practically says it runs only in initial setup. Is that really the right parent to use? I'd expect only `NormalSpoke` here...

Reply:
Initial setup has not been touched for a long time, is hardly a priority, and we're asking around to see if we can just drop it completely. So I'd expect it to just keep its own eula spoke and this be in anaconda only.

Reply:
Yeah, that might make more sense - not that much code is needed for EULA display, so it might make sense to just duplicate it in Anaconda, rather than to keep the screen reusable, with that special mixin and the necessary Initial Setup changes.